### PR TITLE
020_HMI_Does_Not_Respond_WRONG_LANGUAGE_RAI_steeringWheelLocation

### PR DESCRIPTION
**Pull request by istoimenova**
Used common functions are implemented in
#603 
Script is part of TS: #597

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

File: test_scripts/API/RAI/020_ATF_N_HMI_Does_Not_Respond_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	
+	EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.registerAppInterfaceParams.appName }})
+	EXPECT_RESPONSE(CorIdRegister, { success = true, resultCode = "WRONG_LANGUAGE", hmiCapabilities = { steeringWheelLocation = value_steering_wheel_location } })
+	EXPECT_NOTIFICATION("OnHMIStatus", {systemContext="MAIN", hmiLevel="NONE", audioStreamingState="NOT_AUDIBLE"} )
+end
+
+--[[ Postconditions ]]
+commonFunctions:newTestCasesGroup("Postconditions")
+
+function Test.Postcondition_Restore_hmi_capabilities()
```
Comment (OVikhrov) [inline test_scripts/API/RAI/020_ATF_N_HMI_Does_Not_Respond_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua]: @istoimenova hmi_capabilities.json was not changed. Script will be aborted on this step

Comment (istoimenova) [inline test_scripts/API/RAI/020_ATF_N_HMI_Does_Not_Respond_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua]: @OVikhrov, done in 6ad7a986d5bd6cad9f27b09bd9d2fe3584a9d75d

**Comment by istoimenova** [comment]
@OVikhrov, please check updates in 6ad7a986d5bd6cad9f27b09bd9d2fe3584a9d75d

**Pull request merged**